### PR TITLE
Mar 1297 download resumes

### PR DESCRIPTION
**Pull request by ndavison**

### CODE REVIEW

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    )
+  }
+
+  render() {
+    return (
+      <span>
+        <strong>
+          Downloading documents for brief &quot;{this.props.brief.title}&quot;...{' '}
+        </strong>
+        {this.props.briefDocumentsData ? this.createDownloadElement() : <LoadingIndicatorFullPage />}
```
Comment (matt-sm) [inline apps/marketplace/components/Brief/BriefDownloadDocuments.js]: Looks a bit odd have the "Downloading.." text with the loading indicator

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      </span>
+    )
+  }
+}
+
+BriefDownloadDocuments.propTypes = {
+  brief: PropTypes.object.isRequired,
+  downloadBriefDocuments: PropTypes.func.isRequired
+}
```
Comment (matt-sm) [inline apps/marketplace/components/Brief/BriefDownloadDocuments.js]: `briefDocumentsData` missing?

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import styles from './BriefDownloadDocuments.scss'
+
+const canDownload = () => {
+  const hasDownload = typeof document.createElement('a').download !== 'undefined'
+  const hasObjectURL = typeof window.URL !== 'undefined' && window.URL.createObjectURL !== 'undefined'
```
Comment (marcoalfonso) [inline apps/marketplace/components/Brief/BriefDownloadDocuments.js]: Don't use the window object, that is vanilla javascript and not the react way. If you want get the url you can use the match prop. Just pass the props to the component and then you can call match.url

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+  componentDidMount() {
+    this.props.downloadBriefDocuments(this.props.brief.id)
+  }
+
+  createDownloadElement() {
+    if (canDownload()) {
+      return (
+        <a
+          className={styles.hidden}
+          href={window.URL.createObjectURL(this.props.briefDocumentsData)}
```
Comment (marcoalfonso) [inline apps/marketplace/components/Brief/BriefDownloadDocuments.js]: Also use match not window. Anti react pattern

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js (hunk 3 above)
Comment (ndavison) [reply apps/marketplace/components/Brief/BriefDownloadDocuments.js]: @marcoalfonso this is to use the Object URL API, it's not accessing the URL of the page (first it's a feature compatibility check, and the next is the actual usage). This is a browser API - is there a better way to bring it in? I may be able to reference it simply as `URL` but either way it's just as global as `window.URL`.

File: apps/marketplace/components/Brief/BriefDownloadDocuments.js (hunk 4 above)
Comment (ndavison) [reply apps/marketplace/components/Brief/BriefDownloadDocuments.js]: Same as above.

**Pull request merged**